tools/prune-old-builds.py

```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+FOURTEEN_DAYS = 14
+
+
+def how_old(date: str) -> int:
+    """Return a integer that represents how many days has passed given a date"""
+    return (datetime.now() - datetime.strptime(date, '%Y-%m-%d')).days
+
+
+if __name__ == '__main__':
+    builds_path = Path('builds/')
+    repo_url = 'https://github.com/quick-lint/quick-lint-js.git'
+    repo_name = 'quick-lint-js'
+
+    if os.path.exists(repo_name):
+        shutil.rmtree(repo_name)
+
+    subprocess.run(['git', 'clone', '-q', repo_url])
+
+    print('Getting all commits of the repository')
+    commits = subprocess.getoutput(
+        f"cd {repo_name} && git rev-list --all --remotes").split('\n')
+
+    builds_path = Path('builds/')
+
+    for folder in os.listdir(builds_path):
```

Comment:
It looks like this code runs on the server. How do you think we should deploy and run this script?

Reply:
We could run this script using cron or keep this script in the server and run it when the pipeline get triggered through ssh.

Reply:
That makes sense.

I guess updates to this script need to be manually deployed (i.e. copied from my machine to the server). I think that's fine.